frontend/src/index.tsx

```diff
@@ -48,13 +66,24 @@ const router = createHashRouter([
       {
         index: true,
         Component: ReportDataHandler,
-        loader: async () => fetchReportData(Dialect.forPath("draft2020-12")),
+        loader: async () => {
+          const reportData = await fetchReportData(
+            Dialect.forPath("draft2020-12"),
+          );
+          const allImplementationsData = await fetchImplementationStatsData();
+          return { reportData, allImplementationsData };
+        },
       },
       {
         path: "/dialects/:draftName",
         Component: ReportDataHandler,
-        loader: async ({ params }) =>
-          fetchReportData(Dialect.forPath(params.draftName!)),
+        loader: async ({ params }) => {
+          const reportData = await fetchReportData(
+            Dialect.forPath(params.draftName!),
+          );
+          const allImplementationsData = await fetchImplementationStatsData();
```

Comment:
These API calls are sequential but could be run in parallel. Also the 2 loader functions are almost exactly the same - as this is only 5 lines long I don't really care, but if you want you can extract the common code